modules/dqt/js/arrayintersect.js

```diff
@@ -102,12 +102,15 @@ export const getSessions = function(group) {
 };
 
 export const enumToArray = function(enumString) {
-    let tempArray = enumString.split('(')[1].split('\'');
-    let array = [];
-    for (let i = 1; i < tempArray.length; i += 2) {
-        array.push(tempArray[i]);
-    }
-    return array;
+  enumString = enumString.substring('enum('.length); // remove 'enum('
```

Comment:
@ridz1208 the modified code in this function fixes the "Filters - Project menu breaks if one of them has apostrophe in the title" bug.